crypto/signer.go

```diff
@@ -20,23 +20,23 @@ const (
 	mapKeyTreeSize       string = "TreeSize"
 )
 
-// TrillianSigner is responsible for signing log-related data and producing the appropriate
+// Signer is responsible for signing log-related data and producing the appropriate
```

Comment:
Yes, but external code will always use the package name, so there's no ambiguity; anything that imports both golang `crypto` and Trillian `crypto` will need to use a renaming import anyway, so there would be (say) `gocrypto.Signer` and `crypto.Signer`.

Cf. https://blog.golang.org/package-names
"Types in different packages can have the same name, because from the client's point of view such names are discriminated by the package name. For example, the standard library includes several types named Reader, including jpeg.Reader, bufio.Reader, and csv.Reader. Each package name fits with Reader to yield a good type name."